packages/validation/README.md

```diff
@@ -0,0 +1,31 @@
+# @lionweb/validation
+
+The code in this package implements the checks to run the tests in `lionweb-integration-testing/testset`.
+
+This package is intentionally kept very lightweigt, e.g. it uses type instead of classes and has has zero dependencies on external libraries/packages.
+
+It assumes that node.js 18 is used, but will propably work with other versions as well.
+
+To run the tests checkout the `lionweb-integration-testing` project in next to the `lionore-typescript` project
+and run
+
+    # install ts-node
+    yarn
```

Comment:
Can we use NPM here as well? `yarn` would obviously work as well, but it would mean listing another tooling dependency.

Reply:
I'll test whether npm works as well and change this if it does.

Reply:
changed all yarn commands to npm, work as well.